public/javascripts/SVLabel/src/SVLabel/util/UtilitiesSidewalk.js

```diff
@@ -269,6 +361,38 @@ function UtilitiesMisc (JSON) {
                 shortcut: {
                     keyNumber: 83,
                     keyChar: 'S'
+                },
+                tagInfo: {
+                    'bumpy': {
+                        keyNumber: 80,
+                        keyChar: 'P',
+                        text: 'bum<tag-underline>p</tag-underline>y',
+                        id: 'bumpy'
+                    },
+                    'uneven': {
+                        keyNumber: 85,
+                        keyChar: 'U',
+                        text: '<tag-underline>u</tag-underline>neven',
+                        id: 'uneven'
+                    },
+                    'cracks': {
+                        keyNumber: 82,
+                        keyChar: 'R',
+                        text: 'c<tag-underline>r</tag-underline>acks',
+                        id: 'cracks'
+                    },
+                    'grass': {
+                        keyNumber: 71,
+                        keyChar: 'G',
+                        text: '<tag-underline>g</tag-underline>rass',
```

Comment:
Well... it is, it's just not very visible huh 😅

Reply:
Yeah I'm not sure why that is but it looks inconsistent.